### PR TITLE
[Feature] Add configs for AOT-GAN

### PR DESCRIPTION
**Pull request by quincylin1**
added aotgan config

### CODE REVIEW

File: configs/inpainting/AOT-GAN/AOT-GAN_512x512_places.py [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+evaluation = dict(interval=50000)
+
+total_iters = 500002
+dist_params = dict(backend='nccl')
+log_level = 'INFO'
+work_dir = './workdirs/aotgan_places'
+load_from = None
+resume_from = None
+workflow = [('train', 10000)]
+exp_name = 'AOT-GAN_512x512_places'
```
Comment (ckkelvinchan) [inline configs/inpainting/AOT-GAN/AOT-GAN_512x512_places.py:187]: We have a naming convention for the models. Would you mind use the convention below?
```
{model}_[model setting]_{backbone}_[refiner]_[norm setting]_[misc]_[gpu x batch_per_gpu]_{schedule}_{dataset}

```
An example can also be found [here](https://github.com/open-mmlab/mmediting/blob/master/configs/inpainting/partial_conv/pconv_256x256_stage1_8x1_celeba.py#L139)

**Review by ckkelvinchan — COMMENTED** [review]
Hi @quincylin1 , would you mind also providing the weights converted from the official weights?

File: configs/inpainting/AOT-GAN/AOT-GAN_512x512_places.py (hunk 1 above)
Comment (ckkelvinchan) [reply configs/inpainting/AOT-GAN/AOT-GAN_512x512_places.py:187]: > Hi @quincylin1 , would you mind also providing the weights converted from the official weights?

The checkpoint can be named as follows:
```
{config_name}_{date}-{hash}.pth
```
For the hash, you can use `tools/publish_model.py` to generate.

Comment (quincylin1) [reply configs/inpainting/AOT-GAN/AOT-GAN_512x512_places.py:187]: > We have a naming convention for the models. Would you mind use the convention below?
> 
> ```
> {model}_[model setting]_{backbone}_[refiner]_[norm setting]_[misc]_[gpu x batch_per_gpu]_{schedule}_{dataset}
> ```
> 
> An example can also be found [here](https://github.com/open-mmlab/mmediting/blob/master/configs/inpainting/partial_conv/pconv_256x256_stage1_8x1_celeba.py#L139)

image

@ckkelvinchan just wondering if this name is ok for this config: `AOT-GAN_512x512_4x12_places.py`

Comment (ckkelvinchan) [reply configs/inpainting/AOT-GAN/AOT-GAN_512x512_places.py:187]: Yeah I think it works.

File: tests/test_models/test_backbones/test_vfi_backbones/test_tof_vfi_net.py [hunk 2]
```diff
@@ -28,11 +28,11 @@ def test_tof_vfi_net():
         model = model.cuda()
         inputs = inputs.cuda()
         output = model(inputs)
-        output = model(inputs, True)
         assert torch.is_tensor(output)
-        assert output.shape == (1, 3, 256, 256)
```
Comment (ckkelvinchan) [inline tests/test_models/test_backbones/test_vfi_backbones/test_tof_vfi_net.py:33]: Is it a bug from previous PR? @Yshuo-Li

**Comment by ckkelvinchan** [comment]
By the way, can @wangruohui or @Yshuo-Li help with the checkpoint uploading?

@quincylin1 Are you able to provide us with the
1. converted official weights
2. weights trained with your code and the corresponding log?

Thanks!

**Comment by quincylin1** [comment]
> Hi @quincylin1 , would you mind also providing the weights converted from the official weights?

Hi @ckkelvinchan, slightly confused here, what do you mean by weights converted from the official weights?

**Comment by wangruohui** [comment]
Hello @quincylin1 , 

Any updates on the model weights?

**Review by Yshuo-Li — APPROVED** [review]

File: configs/inpainting/AOT-GAN/metafile.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  README: configs/inpainting/AOT-GAN/README.md
+Models:
+- Config: configs/inpainting/AOT-GAN/AOT-GAN_512x512_4x12_places.py
+  In Collection: AOT-GAN
+  Metadata:
+    Training Data: PLACES
+  Name: AOT-GAN_512x512_4x12_places
+  Results:
+  - Dataset: PLACES
+    Metrics: {}
```
[the remark this answers is not in the document]
Comment (wangruohui) [reply configs/inpainting/AOT-GAN/metafile.yml]: Fixed

**Pull request merged**